Base/usr/share/man/man2/setresuid.md

```diff
@@ -0,0 +1,38 @@
+## Name
+
+setresuid, setresgid - set real, effective, and saved user / group id
+
+## Synopsis
+
+```**c++
+#include <unistd.h>
+
+int setresuid(uid_t, uid_t, uid_t);
+int setresgid(gid_t, gid_t, gid_t);
+```
+
+## Description
+
+Sets all of real, effective, and saved user or group id to the given values.
+
+`-1` means "no change".
```

Comment:
"If one or more of the values is set to -1, the corresponding ID is not changed by this syscall.", or something like that.

Reply:
Tweaked.